README.md

```diff
@@ -167,5 +196,13 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+goldlamp = Lamp.create(wishes_remaining: 3)
+genie = Genie.create(name: "Genie", lamp_id: 1)
+
+goldlamp.update(wishes_remaining: 1)
+
+blacklamp = Lamp.create(wishes_remaining: 3)
+jafar = Genie.create(name: "Jafar", lamp_id: 2)
+
+goldlamp.destroy
```

Comment:
close! I don't think we want to necessarily destroy the gold lamp in this example but instead we just want to update it to be equal to nil:

`genie.update(lamp: nil)`